packages/core/src/_common/base/login.js

```diff
@@ -45,6 +46,12 @@ const Login = (() => {
         if (getAppId() === domain_app_ids['app.deriv.com'] && /^app\.deriv\.com$/.test(window.location.hostname)) {
             return default_login_url;
         }
+        if (
+            getAppId() === domain_app_ids['deriv-crypto.com'] &&
+            /^(www\.)?deriv-crypto\.com$/.test(window.location.hostname)
```

Comment:
```suggestion
            /^app\.deriv-crypto\.com$/.test(window.location.hostname)
```

Reply:
let's move this to a function in shared as well we'll need it in other places as well

Reply:
Moved `loginUrl` to shared, since it required `storage` as well, had to move that one too. Since the login itself requires `ClientBase`, I think it would be better if we do that refactor in another PR. Since the `logged_in` status should be passed from the consumer side and another monorepo wide refactor is required.

Reply:
Sure, i'm removing `ClientBase` in a different PR can handle it there